cub/cmake/cub-config-version.cmake

```diff
@@ -1,5 +1,5 @@
 # Parse version information from version.cuh:
-file(READ "${CMAKE_CURRENT_LIST_DIR}/../version.cuh" CUB_VERSION_HEADER)
```

Comment:
File extension needs to stay `.cuh`

Reply:
Woops, maybe that's why things broke for me when changing this 😅